test/XRP/issued-currency-client.test.ts

```diff
@@ -0,0 +1,22 @@
+import { XrplNetwork } from 'xpring-common-js'
+import IssuedCurrencyClient from '../../src/XRP/issued-currency-client'
+import { FakeXRPNetworkClient } from './fakes/fake-xrp-network-client'
+import 'mocha'
+
+const fakeSucceedingNetworkClient = new FakeXRPNetworkClient()
+// const fakeErroringNetworkClient = new FakeXRPNetworkClient(
+//   FakeXRPNetworkClientResponses.defaultErrorResponses,
+// )
```

Comment:
what's this for

Reply:
that's going to be needed shortly for the unit tests so I left the infra in